bazel/upb_proto_library.bzl

```diff
@@ -175,7 +175,7 @@ def _upb_proto_rule_impl(ctx):
              "_WrappedGeneratedSrcsInfo (aspect should have handled this).")
     cc_info = dep[_WrappedCcInfo].cc_info
     srcs = dep[_WrappedGeneratedSrcsInfo].srcs
-    lib = cc_info.linking_context.libraries_to_link.to_list()[0]
+    lib = cc_info.linking_context.libraries_to_link[0]
```

Comment:
`to_list()` should be working I think? It appears to exist way back to Bazel 0.20 and before. What breaks when you have `to_list()` in there?

Reply:
That doesn't work with bazel 0.26, no. It says that `linking_context.libraries_to_link` doesn't have it `to_list()`:

```
type 'list' has no method to_list()
```

Reply:
Managed to wrangle some code to make it work pre and post bazel 0.27.